src/helpers/class-general-helper.php

```diff
@@ -23,6 +23,8 @@ class General_Helper {
    * @return string       Value of the key if it exists, empty string if not.
    *
    * @since 1.0.0
+   *
+   * // TODO: Refactor and remove.
    */
   public static function get_array_value( $key, $array ) {
```

Comment:
I thought that we said this wasn't needed? We'd use null coalesce operator instead (`??`) or were there some cases where we couldn't use this.

CC @IvanGrginovInf

Reply:
@dingo-d Yup, hence the `to do` I assume. (refactor all places where this is used and remove).

Reply:
removed